src/utils/index.ts

```diff
@@ -12,7 +12,6 @@ export function getScrollbarSize(parentNode = document.body) {
   outer.setAttribute('data-measure-scrollbar', 'true');
   outer.style.visibility = 'hidden';
   outer.style.overflow = 'scroll';
-  outer.style.msOverflowStyle = 'scrollbar';
```

Comment:
why remove this?

Reply:
![](https://i.rawr.dev/DWrSK7vcPq.png)
It was giving out some compiling issues, not sure if it's my setup